Makefile.am

```diff
@@ -18,7 +19,11 @@ BUILT_SOURCES = \
 BUILD_VERSION = `git describe --abbrev=0 --dirty --always --tags`
 
 AM_TESTS_ENVIRONMENT = \
-export LD_LIBRARY_PATH=$(LIBRARY_PATH);
+	LD_LIBRARY_PATH=$(INSTALL_DIR)/lib
```

Comment:
Alternatively, this could be set in the caller's environment (where `make check` is called), but I'd say that when we exterminate the dynamic libraries from `$(INSTALL_DIR)/lib`, we'll just ding all the references to `LD_LIBRARY_PATH`, and forget all about that particular bad dream...